tests/libraries/token/test_standard_token.py

```diff
@@ -2,12 +2,25 @@
 
 from ethereum.tools import tester
 from ethereum.tools.tester import TransactionFailed
-from pytest import raises
+from pytest import fixture, mark, lazy_fixture, raises
 
+@fixture(scope='session')
+def testerSnapshot(sessionFixture):
+    sessionFixture.resetSnapshot()
+    sessionFixture.uploadAndAddToController("solidity_test_helpers/StandardTokenHelper.sol")
+    standardToken = sessionFixture.contracts['StandardTokenHelper']
+    return sessionFixture.chain.snapshot()
 
-def test_eternal_approval_magic(contractsFixture):
+
+@fixture
+def testerContractsFixture(sessionFixture, testerSnapshot):
```

Comment:
Consider coming up with a more pithy name for this fixture.  `testerContractsFixture` doesn't really imply what it is used for.  Also, since this is a local fixture feel free to use something shorter than normal like `fixture` (unfortunately that name is taken though).

Reply:
Maybe `tokenFixture`?